include/oneapi/dpl/internal/async_impl/glue_async_impl.h

```diff
@@ -133,9 +133,9 @@ reduce_async(_ExecutionPolicy&& __exec, _ForwardIterator __first, _ForwardIterat
 {
     using _ValueType = typename ::std::iterator_traits<_ForwardIterator>::value_type;
```

Comment:
As far as `_ValueType` is not used anymore, this line can be removed as well.

Reply:
good point, fixed - could you please approve now?